### ISSUE TITLE
Matcher "nilify_blanks" doesn't filter columns by type, causes failure

### ISSUE THREAD

**Issue by mroach** [rubiety/nilify_blanks#22]
The `nilify_blanks` matcher runs this check to ensure the list of columns to nilify is correct:

```ruby
model_class.nilify_blanks_columns == model_class.content_columns.select(&:null).map(&:name).map(&:to_s)
```

However, `nilify_blanks_columns` when initialised without any explicit columns loads the column list like so:
```ruby
self.nilify_blanks_columns = self.content_columns.select(&:null).select {|c| options[:types].include?(c.type) }.map(&:name).map(&:to_s)
```

The issue then is that the matcher is NOT filtering by column types like the initialiser is. In my case, my model has a date field that is not included in `nilify_blanks_columns` (because the default types are only `:string` and `:text`) so my spec fails.

My temporary workaround, which is fine because I'd like everything to be NULL, is initialising with:

```ruby
ActiveRecord::Base.nilify_blanks types: [:string, :text, :date, :time, :datetime, :inet]
```

**Comment by rubiety** [comment]
@mroach Good catch - thanks! Indeed this was an inconsistency. This should be fixed in commit 62c958d pushed to master. If you want to give your app a try against that to verify this fixed worked for you, feel free. Otherwise this will be in the next gem release.

**Comment by abelmartin** [comment]
Hi there.  I wanted to know if you could cut 1.2.1 which includes would include this update to the matchers.  I'd really appreciate it. :smile: :+1:

**Comment by rubiety** [comment]
Okay, just released `1.2.1`. Enjoy!

https://rubygems.org/gems/nilify_blanks/versions/1.2.1

**Comment by abelmartin** [comment]
Thanks so much :smile:
